plugin/src/io/github/alexarchambault/millnativeimage/NativeImage.scala

```diff
@@ -316,10 +317,7 @@ object NativeImage {
   }
 
   def vcvarsOpt: Option[os.Path] = {
-    val workspace = sys.env.get("MILL_WORKSPACE_ROOT") match {
-      case Some(dir) => os.Path(dir)
-      case None => os.pwd
-    }
+    val workspace = WorkspaceRoot.workspaceRoot.getOrElse(os.pwd)
```

Comment:
This doesn't work on Mill `0.10`. We could use it with version-specific sources.
Let me know what you prefer and I can implement it.

Reply:
I reverted it, version-specific sources would be an overkill here, I think